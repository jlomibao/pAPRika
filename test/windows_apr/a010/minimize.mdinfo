

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2750E+02     4.8986E-01     1.5809E+00     H32       105

 BOND    =        1.2648  ANGLE   =      107.5398  DIHED      =       52.6182
 VDWAALS =      -68.5972  EEL     =     1332.4170  EGB        =     -127.4311
 1-4 VDW =        5.5117  1-4 EEL =    -2130.8438  RESTRAINT  =        0.0174
 EAMBER  =     -827.5207
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================
