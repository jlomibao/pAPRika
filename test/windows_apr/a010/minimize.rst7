defa
  125
  -0.5297429   4.9580127   1.8215462   1.0066888   4.8950869   1.8221948
  -0.8903136   4.2815778   0.5828678  -0.8949213   4.2465007   3.0387364
  -0.9094579   5.9861576   1.8349960   1.4692453   5.8887116   1.8362520
   1.3119549   4.1916097   0.5839243   1.3118860   4.1556372   3.0391712
   2.6803850   4.0765955   0.1004031   2.6753509   4.0929976   3.5513609
   0.1928709   3.8254591   3.7587026   0.1969015   3.8873550  -0.1505082
   0.1727232   3.3267171   4.8706533   0.1789241   3.4224203  -1.2769722
   2.6649485   4.0262184  -0.9944331   3.2421487   4.9593757   0.4268484
   2.6387622   4.0632077   4.6464483   3.2162867   4.9867635   3.2196781
  -4.6604232   1.9852633   1.8300178  -3.8187747   3.2728460   1.8265137
  -4.2265676   1.2983768   0.6210222  -4.2753891   1.3392943   3.0777991
  -5.7377396   2.1858759   1.8057245  -4.4352622   4.1788423   1.8028461
  -3.0220104   3.1475085   0.6135029  -3.0666112   3.1828897   3.0718649
  -2.2629205   4.2787738   0.0972567  -2.2588249   4.2964944   3.5510056
  -3.3896304   2.0788140   3.8150463  -3.3019814   2.0099746  -0.0973826
  -3.0147056   1.8369170   4.9494185  -2.8695644   1.7245089  -1.2006025
  -2.2500624   4.2245902  -0.9974311  -2.7502793   5.2057733   0.4206750
  -2.2249060   4.2621459   4.6460454  -2.7237862   5.2324891   3.2205957
  -4.0946939  -3.0772427   1.8559389  -4.8218305  -1.7216684   1.8469280
  -3.3318279  -3.0543414   0.6143197  -3.2945806  -3.0185449   3.0720057
  -4.7868850  -3.9268393   1.8783770  -5.9126343  -1.8281404   1.8629590
  -4.3724882  -1.1113985   0.6025364  -4.3386101  -1.0743279   3.0590536
  -4.9485305   0.1359532   0.1200891  -4.9606945   0.1464270   3.5555051
  -3.4802726  -1.8609640   3.7814295  -3.5478546  -1.9242924  -0.1283166
  -3.0291414  -1.6107604   4.8857222  -3.1406424  -1.7124967  -1.2575770
  -4.9097535   0.1422488  -0.9753101  -5.9892606   0.1951784   0.4589389
  -4.9304114   0.1367157   4.6511470  -6.0001957   0.1774344   3.2092078
   0.5782999  -5.0996320   1.8686723  -0.9581262  -5.0354141   1.8687819
   0.9426625  -4.3940093   0.6478968   0.9407567  -4.4155123   3.1026176
   0.9577250  -6.1279380   1.8603190  -1.4220272  -6.0284797   1.8608530
  -1.2628363  -4.3021054   0.6480056  -1.2618152  -4.3231653   3.1026874
  -2.6262490  -4.2359062   0.1376939  -2.6295539  -4.2073673   3.5889836
  -0.1455376  -4.0128545   3.8329522  -0.1444152  -3.9711062  -0.0714950
  -0.1260089  -3.5366003   4.9546354  -0.1241250  -3.4684821  -1.1817422
  -2.5914351  -4.2083868  -0.9575108  -3.1702123  -5.1266740   0.4724815
  -2.6114754  -4.1532738   4.6836021  -3.1912968  -5.0918226   3.2670677
   4.7079069  -2.1240842   1.8477940   3.8678863  -3.4127024   1.8561627
   4.3114545  -1.4766605   0.6043220   4.2821483  -1.4386680   3.0605295
   5.7856839  -2.3231034   1.8631048   4.4851224  -4.3182523   1.8772649
   3.1086905  -3.3236177   0.6152880   3.0764984  -3.2870978   3.0728426
   2.3066629  -4.4413345   0.1372166   2.3132579  -4.4156230   3.5893313
   3.3604812  -2.1500107   3.7829513   3.4193150  -2.2150779  -0.1262543
   2.9325144  -1.8641901   4.8878216   3.0297662  -1.9686063  -1.2547070
   2.2740359  -4.4094452  -0.9579369   2.7742772  -5.3749718   0.4705397
   2.2994916  -4.3605126   4.6839617   2.7984494  -5.3442557   3.2673427
   4.1458901   2.9395453   1.8289178   4.8698340   1.5820725   1.8323069
   3.3428306   2.8853849   0.6147695   3.3885976   2.9156842   3.0742559
   4.8404489   3.7871969   1.8060954   5.9607305   1.6860905   1.8084079
   4.3769872   0.9364800   0.6229934   4.4284158   0.9719201   3.0795485
   4.9938663  -0.2845475   0.1212650   5.0068136  -0.2761865   3.5578506
   3.6103637   1.7861869   3.8163184   3.5182292   1.7265892  -0.0949948
   3.2135377   1.5761544   4.9495818   3.0604065   1.4783085  -1.1969624
   4.9546203  -0.2748122  -0.9740923   6.0361643  -0.3166244   0.4589509
   4.9744384  -0.2832898   4.6534527   6.0455737  -0.3354260   3.2130113
   0.0144340  -0.0531572  -0.0653446  -0.9013322  -0.6344655  -0.2285507
   0.8743634  -0.7128804  -0.2340597   0.0476371   0.7498071  -0.8118911
   0.0442263   0.5236887   1.3612006   0.9538019   1.1225011   1.4926948
  -0.0023161  -0.5913799   2.4259518  -0.8078949   1.2007500   1.4978632
  -0.9118346  -1.1901697   2.2939446   0.8497026  -1.2685158   2.2890205
   0.0273417  -0.0156957   3.8530282  -0.8321597   0.6445424   4.0219145
  -0.0069045  -0.8196491   4.5984614   0.9436389   0.5645252   4.0171339
   0.0006786  -0.0003937  -6.0018004  -0.0006257  -0.0013624  -9.0000676
  -0.0001001   2.2000035 -11.2000229
