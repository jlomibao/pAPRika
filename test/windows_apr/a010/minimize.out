
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:06:56

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/a010
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   603753 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -8.1356E+02     1.0057E+01     2.4770E+01     N12        57

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -67.7110  EEL     =     1326.2731  EGB        =     -123.8292
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -813.5649
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.2743E+02     4.9765E-01     1.6641E+00     H7        120

 BOND    =        1.2509  ANGLE   =      107.5596  DIHED      =       52.5872
 VDWAALS =      -68.4410  EEL     =     1332.0537  EGB        =     -127.3762
 1-4 VDW =        5.4825  1-4 EEL =    -2130.5539  RESTRAINT  =        0.0078
 EAMBER  =     -827.4372
 NMR restraints: Bond =    0.003   Angle =     0.005   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.2746E+02     4.9322E-01     1.5808E+00     H32       105

 BOND    =        1.2574  ANGLE   =      107.5466  DIHED      =       52.6135
 VDWAALS =      -68.5349  EEL     =     1332.3044  EGB        =     -127.4124
 1-4 VDW =        5.5043  1-4 EEL =    -2130.7494  RESTRAINT  =        0.0118
 EAMBER  =     -827.4705
 NMR restraints: Bond =    0.003   Angle =     0.008   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.2748E+02     4.9681E-01     1.5807E+00     H32       105

 BOND    =        1.2503  ANGLE   =      107.5448  DIHED      =       52.6151
 VDWAALS =      -68.5632  EEL     =     1332.3453  EGB        =     -127.4019
 1-4 VDW =        5.5103  1-4 EEL =    -2130.7922  RESTRAINT  =        0.0138
 EAMBER  =     -827.4916
 NMR restraints: Bond =    0.004   Angle =     0.009   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.2749E+02     4.9074E-01     1.5808E+00     H32       105

 BOND    =        1.2639  ANGLE   =      107.5418  DIHED      =       52.6158
 VDWAALS =      -68.5789  EEL     =     1332.3930  EGB        =     -127.4303
 1-4 VDW =        5.5098  1-4 EEL =    -2130.8203  RESTRAINT  =        0.0152
 EAMBER  =     -827.5053
 NMR restraints: Bond =    0.004   Angle =     0.010   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.2750E+02     4.9025E-01     1.5809E+00     H32       105

 BOND    =        1.2635  ANGLE   =      107.5407  DIHED      =       52.6168
 VDWAALS =      -68.5898  EEL     =     1332.4063  EGB        =     -127.4294
 1-4 VDW =        5.5111  1-4 EEL =    -2130.8340  RESTRAINT  =        0.0164
 EAMBER  =     -827.5146
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.2750E+02     4.8986E-01     1.5809E+00     H32       105

 BOND    =        1.2648  ANGLE   =      107.5397  DIHED      =       52.6182
 VDWAALS =      -68.5971  EEL     =     1332.4167  EGB        =     -127.4311
 1-4 VDW =        5.5117  1-4 EEL =    -2130.8434  RESTRAINT  =        0.0174
 EAMBER  =     -827.5205
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.2750E+02     4.8986E-01     1.5809E+00     H32       105

 BOND    =        1.2648  ANGLE   =      107.5398  DIHED      =       52.6182
 VDWAALS =      -68.5972  EEL     =     1332.4168  EGB        =     -127.4311
 1-4 VDW =        5.5117  1-4 EEL =    -2130.8435  RESTRAINT  =        0.0174
 EAMBER  =     -827.5206
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.2750E+02     4.8986E-01     1.5809E+00     H32       105

 BOND    =        1.2647  ANGLE   =      107.5398  DIHED      =       52.6182
 VDWAALS =      -68.5972  EEL     =     1332.4168  EGB        =     -127.4311
 1-4 VDW =        5.5118  1-4 EEL =    -2130.8435  RESTRAINT  =        0.0174
 EAMBER  =     -827.5206
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.2750E+02     4.8986E-01     1.5809E+00     H32       105

 BOND    =        1.2647  ANGLE   =      107.5397  DIHED      =       52.6182
 VDWAALS =      -68.5972  EEL     =     1332.4168  EGB        =     -127.4311
 1-4 VDW =        5.5118  1-4 EEL =    -2130.8435  RESTRAINT  =        0.0174
 EAMBER  =     -827.5206
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2750E+02     4.8986E-01     1.5809E+00     H32       105

 BOND    =        1.2648  ANGLE   =      107.5398  DIHED      =       52.6182
 VDWAALS =      -68.5972  EEL     =     1332.4170  EGB        =     -127.4311
 1-4 VDW =        5.5117  1-4 EEL =    -2130.8438  RESTRAINT  =        0.0174
 EAMBER  =     -827.5207
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2750E+02     4.8986E-01     1.5809E+00     H32       105

 BOND    =        1.2648  ANGLE   =      107.5398  DIHED      =       52.6182
 VDWAALS =      -68.5972  EEL     =     1332.4170  EGB        =     -127.4311
 1-4 VDW =        5.5117  1-4 EEL =    -2130.8438  RESTRAINT  =        0.0174
 EAMBER  =     -827.5207
 NMR restraints: Bond =    0.005   Angle =     0.011   Torsion =     0.001
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.305    9.302    0.003    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.270  147.398    0.128    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.489   49.483    0.006    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   61.995   62.003    0.008    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.057 -146.178    0.121    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.702  -72.548    0.154    0.001 t
  DUM  DM1    3 --  C    BUT    2:    5.937    6.000    0.063    0.005 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.461  180.000    0.539    0.002 a
  DUM  DM1    3 --  C3   BUT    2:  178.942  180.000    1.058    0.008 a
                                       Total distance penalty:      0.005
                                       Total angle    penalty:      0.011
                                       Total torsion  penalty:      0.001
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.465
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.33 (22.47% of Gen B)
|                Calc gb diag               4.10 (39.50% of Gen B)
|                Calc gb off-diag           3.94 (38.01% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.38 (100.0% of Nonbo)
|          Nonbond force             10.38 (90.67% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.04% of Force)
|          Other                      0.03 ( 0.29% of Force)
|       Force time                11.44 (100.0% of Runmd)
|    Runmd Time                11.44 (99.76% of Total)
|    Other                      0.03 ( 0.24% of Total)
| Total time                11.47 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:06:56.599  on 04/10/2019
|           Setup done at 18:06:56.605  on 04/10/2019
|           Run   done at 18:07:08.072  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
